sr_multi_moveit/sr_box_ur10_moveit_config/config/robot_configs/left_ur3.yaml

```diff
@@ -0,0 +1,15 @@
+robot:
+  name: ursrbox
+  manipulators:
+    right_manipulator:
+      side: right
```

Comment:
```suggestion
      side: left
```

Reply:
Fixed in https://github.com/shadow-robot/sr_interface/pull/419/commits/69bc13be8e64f769c2179bda3d2bfc388b4bc071